source/features/conflict-marker.tsx

```diff
@@ -1,96 +1,65 @@
+	const prConfigs = links.map(link => {
+		const [, owner, name, , prNumber] = link.pathname.split('/');
+		const key = api.escapeKey(owner, name, prNumber);
+		return {
+			key, link, owner, name, number: Number(prNumber),
+		};
+	});
 
-function createQueryFragment(pr: PRConfig): string {
-	return `
-		${pr.key}: repository(owner: "${pr.user}", name: "${pr.repo}") {
-			pullRequest(number: ${pr.number}) {
+	// Batch queries cannot be exported to .gql files
+	const batchQuery = prConfigs.map(({key, owner, name, number}) => `
+		${key}: repository(owner: "${owner}", name: "${name}") {
+			pullRequest(number: ${number}) {
 				mergeable
 			}
 		}
-	`;
-}
-
-function buildQuery(prs: PRConfig[]): string {
-	return prs.map(pr => createQueryFragment(pr)).join('\n');
-}
-
-function getPRConfig(prIcon: Element): PRConfig {
-	const link = prIcon.closest('.js-navigation-item')!.querySelector('a.js-navigation-open')!;
-	const [, user, repo, , number] = link.pathname.split('/');
-	return {
-		user,
-		repo,
-		number,
-		link,
-		key: api.escapeKey(user, repo, number),
-	};
-}
-
-async function init(): Promise<false | void> {
-	// Milestone issues are lazy-loaded
-	if (pageDetect.isMilestone()) {
-		// TODO: Use observe instead
-		await oneMutation(select('.js-milestone-issues-container')!, {childList: true});
-	}
-
-	const openPrIcons = select.all('.js-issue-row .octicon-git-pull-request.color-fg-open');
-	if (openPrIcons.length === 0) {
-		return false;
-	}
+	`).join('\n');
 
-	const prs = openPrIcons.map(icon => getPRConfig(icon));
-	const data = await api.v4(buildQuery(prs));
+	const data = await api.v4(batchQuery);
 
-	for (const pr of prs) {
+	for (const pr of prConfigs) {
 		if (data[pr.key].pullRequest.mergeable === 'CONFLICTING') {
 			pr.link.after(
 				<a
 					className="rgh-conflict-marker tooltipped tooltipped-e color-fg-muted ml-2"
 					aria-label="This PR has conflicts that must be resolved"
 					href={`${pr.link.pathname}#partial-pull-merging`}
 				>
-					<AlertIcon className="v-align-middle"/>
-				</a>,
+					<AlertIcon className="v-align-middle" />
+				</a>
 			);
 		}
 	}
 }
 
+function init(signal: AbortSignal): void {
+	observe('.js-issue-row:has(.octicon-git-pull-request.color-fg-open) a.js-navigation-open', batchedFunction(addIcon, {delay: 100}), {signal});
+}
+
 void features.add(import.meta.url, {
 	include: [
 		pageDetect.isIssueOrPRList,
 	],
-	exclude: [
-		pageDetect.isGlobalIssueOrPRList,
-		pageDetect.isBlank,
-	],
-	awaitDomReady: true, // TODO: Use observe + batched-function
-	deduplicate: 'has-rgh-inner', // TODO: Use observe instead
-	init,
-}, {
-	include: [
-		pageDetect.isGlobalIssueOrPRList,
-	],
 	exclude: [
 		pageDetect.isBlank,
 	],
-	deduplicate: 'has-rgh',
-	awaitDomReady: true, // TODO: Use observe + batched-function
 	init,
 });
+
+/*
+Test URLs
+https://github.com/pulls
+https://github.com/refined-github/sandbox/issues?q=conflict
+https://github.com/kubernetes/kubernetes/milestone/62
```

Comment:
This section exists to simplify testing so we should try to use URLs that do and always will include the feature. Our sandbox repo is where these scenarios exist and can be created.